easybuild/easyblocks/c/cplex.py

```diff
@@ -98,20 +86,15 @@ def install_step(self):
 
         run_cmd_qa(cmd, qanda, no_qa=noqanda, log_all=True, simple=True)
 
-        if self.cfg['staged_install']:
-            # move staged installation to actual install dir
-            try:
-                # copytree expects target directory to not exist yet
-                shutil.rmtree(self.installdir)
-                shutil.copytree(stagedir, self.installdir)
-            except OSError, err:
-                self.log.error("Failed to move staged install from %s to %s: %s" % (stagedir, self.installdir, err))
-
         try:
             os.chmod(self.installdir, stat.S_IRWXU | stat.S_IXOTH | stat.S_IXGRP | stat.S_IROTH | stat.S_IRGRP)
         except OSError, err:
             self.log.exception("Can't set permissions on %s: %s" % (self.installdir, err))
 
+    def post_install_step(self):
+        """Determine bin directory for this CPLEX installation."""
+        super(EB_CPLEX, self).post_install_step()
```

Comment:
yes, since `Binary` (i.e., the parent class), now takes care of the `staged_install` stuff